### PR TITLE
update avail integration content

### PR DESCRIPTION
**Pull request by mirnawong1**
updating avialable integration page and migration guide based on updated integration info. 

cc @azzam34

### CODE REVIEW

File: website/docs/guides/migration/sl-migration.md [hunk 1]
```diff
@@ -77,6 +77,11 @@ Now that your Semantic Layer is set up, you will need to update any downstream i
 
 ### Migration guide for Hex
 
+:::important Hex integration coming soon
+
+Hex is currently working on an integration with the new dbt Semantic Layer. This section will have updated instructions once the integration is available.
+:::
+
 1. Set up a new connection for the Semantic Layer for your account. Something to note is that your old connection will still work. The following Loom video guides you in setting up your Semantic Layer with Hex:
```
Comment (mirnawong1) [inline website/docs/guides/migration/sl-migration.md:85]: @rpourzand do we need the steps that follow this "migration guide for hex" section due to the recent update? or is the callout enough right now?

File: website/docs/guides/migration/sl-migration.md [hunk 2]
```diff
@@ -77,6 +77,11 @@ Now that your Semantic Layer is set up, you will need to update any downstream i
 
 ### Migration guide for Hex
 
+:::important Hex integration coming soon
+
+Hex is currently working on an integration with the new dbt Semantic Layer. This section will have updated instructions once the integration is available.
```
Comment (mirnawong1) [inline website/docs/guides/migration/sl-migration.md]:
```suggestion
Hex’s Semantic Layer integration will be available for use in the coming weeks. This section will have updated instructions once the integration is available.
```

File: website/snippets/_sl-partner-links.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <!-- turn this list into sections once more docs are provided -->
 The dbt Semantic Layer integrations are capable of querying dbt metrics, importing definitions, surfacing the underlying data in partner tools, and more.  These are the following tools that integrate with the dbt Semantic Layer:
 
-1. **Hex** &mdash; Public docs are coming soon, refer to [this Loom video](https://www.loom.com/share/752e85aabfbf4fa585008a5598f3517a) for more info on how to integrate with Hex. 
 1. **Mode** &mdash; Refer to the [Mode docs](https://mode.com/help/articles/supported-databases/#dbt-semantic-layer) for info on how to integrate with Mode.
+1. **Hex** &mdash; Hex is currently working on an integration with the new dbt Semantic Layer. Until then, refer to [this Loom video](https://www.loom.com/share/752e85aabfbf4fa585008a5598f3517a) for more info.
```
Comment (mirnawong1) [inline website/snippets/_sl-partner-links.md]:
```suggestion
1. **Hex** &mdash; Hex’s Semantic Layer integration will be available in the coming weeks. Until then, refer to [this Loom video](https://www.loom.com/share/752e85aabfbf4fa585008a5598f3517a) for more info. 
```

File: website/docs/guides/migration/sl-migration.md (hunk 1 above)
Comment (rpourzand) [reply website/docs/guides/migration/sl-migration.md:85]: I think this callout is enough for now

**Review by rpourzand — APPROVED** [review]

**Pull request merged**